pkg/notes/options.go

```diff
@@ -143,6 +152,27 @@ func (o *Options) ValidateAndFinish() error {
 		}
 	}
 
+	// Fail early if the template is broken.
+	if o.Format == FormatOutputMarkdown && o.TemplatePath != "" {
+		// Look in default directory if using default template.
+		if o.TemplatePath == DefaultMarkdownTemplate {
+			cwd, err := os.Getwd()
+			if err != nil {
+				return errors.Wrap(err, "getting current dir")
+			}
+			o.TemplatePath = filepath.Join(cwd, "templates", o.TemplatePath)
+		}
```

Comment:
@saschagrunert See my PR comment. This is making the test fail. I envisioned using the template as a drop-in for notes.RenderMarkdown() so we could delete that method. But at least in testing environment the template isn't copied to the working directory for the binary/test. 

I thought I might have to edit the BUILD.bazel in some way?

Reply:
Hm, I'm not sure can you write the template ad-hoc? We could put the template as inline const string into the .go implementation.

Regarding bazel I'd recommend to run the scripts in hack:
```
> ./hack/update-all.sh
```
then commit the changes. And to verify:
```
> ./hack/verify-all.sh
```

Reply:
I kind of like the template being external. I'll mess around a bit more to see why that test is failing. My best (naive) guess is that `templates/release_notes.md` isn't being copied somehow.

Reply:
Moved to go const based template per you suggestion.